### PR TITLE
[docs][ingress-nginx] Required settings for inlets

### PR DESCRIPTION
**Pull request by meatuses**
## Description

Add footnotes for inlet types with required spec settings.

## Why do we need it, and what problem does it solve?

When setting an inlet type, it's not clear if it has any additional required parameters that should be configured.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: ingress-nginx
type: chore
summary: Minor documentation changes.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @meatuses. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @meatuses. I've started the workflow for label 'deploy/web/test'...

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml [hunk 1]
```diff
@@ -19,10 +19,14 @@ spec:
...
-                    * `HostWithFailover` — устанавливаются два Ingress-контроллера — основной и резервный. Основной контроллер запускается в hostNetwork. Если поды основного контроллера недоступны, трафик уходит в резервный контроллер.
+                      Требуется установка обязательных параметров в секции [spec.HostPort](#ingressnginxcontroller-v1-spec-hostport).
+                    * `HostPortWithProxyProtocol` — устанавливается Ingress-контроллер, который доступен на портах узлов через `hostPort` и использует proxy-protocol для получения настоящего IP-адреса клиента;
+
+                      Требуется установка обязательных параметров в секции [spec.HostPortWithProxyProtocol](#ingressnginxcontroller-v1-spec-hostportwithproxyprotocol).
+
+                      **Внимание!** При использовании этого inlet'а вы должны быть уверены, что запросы к Ingress направляются только от доверенных источников. Одним из способов настройки ограничения может служить опция `acceptRequestsFrom`.
```
Comment (z9r5) [inline modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:28]:
```suggestion
                      **Внимание!** При использовании этого inlet'а вы должны быть уверены, что запросы к Ingress-контроллеру направляются только от доверенных источников. Одним из способов настройки ограничения может служить параметр [acceptRequestsFrom](cr.html#ingressnginxcontroller-v1-spec-acceptrequestsfrom).
```

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml [hunk 2]
```diff
@@ -19,10 +19,14 @@ spec:
...
 
-                      **Внимание!** При использовании этого inlet'а вы должны быть уверены, что запросы к Ingress направляются только от доверенных источников. Одним из способов настройки ограничения может служить опция `acceptRequestsFrom`;
-                    * `HostWithFailover` — устанавливаются два Ingress-контроллера — основной и резервный. Основной контроллер запускается в hostNetwork. Если поды основного контроллера недоступны, трафик уходит в резервный контроллер.
+                      Требуется установка обязательных параметров в секции [spec.HostPort](#ingressnginxcontroller-v1-spec-hostport).
+                    * `HostPortWithProxyProtocol` — устанавливается Ingress-контроллер, который доступен на портах узлов через `hostPort` и использует proxy-protocol для получения настоящего IP-адреса клиента;
+
+                      Требуется установка обязательных параметров в секции [spec.HostPortWithProxyProtocol](#ingressnginxcontroller-v1-spec-hostportwithproxyprotocol).
```
Comment (z9r5) [inline modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:26]: @MariaMakeeva  Тут суть примерно такая  - если выбираешь  HostPortWithProxyProtocol, то обязательно настрой параметры в секции https://deckhouse.io/documentation/v1/modules/402-ingress-nginx/cr.html#ingressnginxcontroller-v1-spec-hostportwithproxyprotocol.

File: modules/402-ingress-nginx/crds/ingress-nginx.yaml [hunk 3]
```diff
@@ -43,8 +43,12 @@ spec:
...
-                    * `HostPort` — Ingress controller is deployed and available through nodes' ports via `hostPort`.
-                    * `HostPortWithProxyProtocol` — Ingress controller is deployed and available through nodes' ports via `hostPort, it uses proxy-protocol to get a real IP of the client.
+                    * `HostPort` — Ingress controller is deployed and available through nodes' ports via `hostPort`;
+
+                      Settings are required in [spec.HostPort](#ingressnginxcontroller-v1-spec-hostport).
+                    * `HostPortWithProxyProtocol` — Ingress controller is deployed and available through nodes' ports via `hostPort, it uses proxy-protocol to get a real IP of the client;
+
+                      Settings are required in [spec.HostPortWithProxyProtocol](#ingressnginxcontroller-v1-spec-hostportwithproxyprotocol).
 
                       **Caution!** Make sure that requests to the Ingress are sent from trusted sources when using this inlet. The `acceptRequestsFrom` parameter can help you with defining trusted sources.
```
Comment (z9r5) [inline modules/402-ingress-nginx/crds/ingress-nginx.yaml:53]:
```suggestion
                      **Caution!** Make sure that requests to the Ingress are sent from trusted sources when using this inlet. The [acceptRequestsFrom](cr.html#ingressnginxcontroller-v1-spec-acceptrequestsfrom) parameter can help you with defining trusted sources.
```

**Review by z9r5 — COMMENTED** [review]
@MariaMakeeva please

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml (hunk 1 above)
Comment (MariaMakeeva) [reply modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:28]: Чуток поправила.

File: modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml (hunk 2 above)
Comment (MariaMakeeva) [inline modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml:26]:
```suggestion
                      Требуется обязательная установка параметров в [секции [spec.HostPort](#ingressnginxcontroller-v1-spec-hostport).
                    * `HostPortWithProxyProtocol` — устанавливается Ingress-контроллер, который доступен на портах узлов через `hostPort` и использует proxy-protocol для получения настоящего IP-адреса клиента;

                      Требуется обязательная установка параметров в [секции spec.HostPortWithProxyProtocol](#ingressnginxcontroller-v1-spec-hostportwithproxyprotocol).
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [7967](https://github.com/deckhouse/deckhouse/pull/7967) to the branch [release-1.58](https://github.com/deckhouse/deckhouse/tree/release-1.58) successful!

**Pull request merged**